Jenkinsfile

```diff
@@ -76,3 +95,15 @@ void withMavenEnv(List envVars = [], def body) {
         body.call()
     }
 }
+
+void renameFiles(def files, String prefix) {
+    for(i = 0; i < files.length; i++) {
```

Comment:
"new File" failed with a security exception, so it's definitely not default whitelisted. (I tried before and it gave me this error which is why I went with the shell method, but I wanted to double check). I'll add a comment about the hackiness